src/Tizen.NUI/src/public/FrameCallbackInterface.cs

```diff
@@ -0,0 +1,254 @@
+    public class FrameCallbackInterface : Disposable
+    {
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal FrameCallbackInterface(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public FrameCallbackInterface() : this(Interop.FrameCallbackInterface.new_FrameCallbackInterface(), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            SwigDirectorConnect();
+        }
+
+        internal static global::System.Runtime.InteropServices.HandleRef getCPtr(FrameCallbackInterface obj)
+        {
+            return (obj == null) ? new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero) : obj.swigCPtr;
+        }
+
+        private void SwigDirectorConnect()
+        {
+            //            if (SwigDerivedClassHasMethod("GetNextFocusableView", swigMethodTypes0))
+            swigDelegate0 = new SwigDelegateFrameCallbackInterface(SwigDirectorOnUpdate);
+            Interop.FrameCallbackInterface.FrameCallbackInterface_director_connect(swigCPtr, swigDelegate0);
+        }
+
+        private bool SwigDerivedClassHasMethod(string methodName, global::System.Type[] methodTypes)
+        {
+            global::System.Reflection.MethodInfo methodInfo = this.GetType().GetMethod(methodName, methodTypes);
+            bool hasDerivedMethod = this.GetType().GetTypeInfo().IsSubclassOf(typeof(FrameCallbackInterface));
+            return hasDerivedMethod && (methodInfo != null);
+        }
+
+        private global::System.IntPtr proxyIntPtr;
+        private void SwigDirectorOnUpdate(global::System.IntPtr proxy, float elapsedSeconds)
+        {
+            proxyIntPtr = proxy;
+            OnUpdate(elapsedSeconds);
+            return;
+        }
+
+        internal delegate void SwigDelegateFrameCallbackInterface(global::System.IntPtr proxy, float elapsedSeconds);
+        private SwigDelegateFrameCallbackInterface swigDelegate0;
+        private static global::System.Type[] swigMethodTypes0 = new global::System.Type[] { typeof(IntPtr), typeof(float) };
+
+
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual void OnUpdate(float elapsedSeconds)
+        {
+
+        }
+
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected bool GetPosition(uint id, Vector3 position)
```

Comment:
I think... even though the following "protected methods" are hidden API, if some comments or some code snips are shown, it will be better. OR you can add some short, simple sample App to show how to use them.
please add it from next time~

Reply:
I will upload test sample to other patch. I will upload it and share it again.